content/security/for-admins/enforce-sign-in/methods.md

```diff
@@ -1,48 +1,94 @@
 ---
-description: Configure registry.json to enforce users to sign into Docker Desktop
-toc_max: 2
-keywords: authentication, registry.json, configure, enforce sign-in
-title: Enforce sign-in for Desktop
-aliases:
-- /docker-hub/configure-sign-in/
+description: Learn about the different ways you can force users to sign in to Docker Desktop
+keywords: authentication, registry.json, configure, enforce sign-in, docker desktop, security 
+title: Ways to enforce sign-in for Docker Desktop
 ---
 
-By default, members of your organization can use Docker Desktop without signing
-in. When users don’t sign in as a member of your organization, they don’t
-receive the [benefits of your organization’s
-subscription](../../subscription/core-subscription/details.md) and they can circumvent [Docker’s
-security features](../../desktop/hardened-desktop/_index.md) for your organization.
+This page outlines the different ways you can enforce sign-in for Docker Desktop.
 
-To ensure members of your organization always sign in, you can deploy a
-`registry.json` configuration file to the machines of your users.
+## Registry key method (Windows OS)
 
-## How is sign-in enforced?
+1. Create the registry key. Your new key should look like the following:
 
-When Docker Desktop starts and it detects a `registry.json` file, the
-following occurs:
+   ```console 
+   $ HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Docker\Docker Desktop
+   ```
+2. Create the string value `allowedOrgs` 
+3. As string data use your organization’s name, all lowercase.
+4. Verify that sign in is enforced. When Docker Desktop starts, verify that the **Sign in required!** prompt appears.
```

Comment:
💡 shall we make more clear Docker Desktop needs to be restarted? I do not want admins to think "When Docker Desktop starts" means opening it by clicking on the icon on the top-right (macOS) or bottom-right (Windows).